python-sdk/telephony-api.md

```diff
@@ -21,20 +24,39 @@ The Python SDK provides the following capabilities:
 - [Subscribe to Events (transcript, questions, action-items, etc.)](#subscribe-to-events)<br/>
 
 
+## Credential Handling 
+
+Symbl.ai's Python SDK simplifies the credential handling by allowing you either to add your credentials directly to the connection method's calls or else through a separate file saved to your execution directory. 
+
+To add your credentials directly to the connection method's calls, add the following line: 
+
+```python
+      credentials={app_id: <app_id>, app_secret: <app_secret>}, #Optional, Don't add this parameter if you have symbl.conf file in your home directory or working directory
+```
+
+To handle credentials through a separate file saved your execution directory, add a file to your project called `symbl.conf` with the following configuration: 
+
+```python 
+[credentials]
+app_id=
+app_secret=
+```
+
 ## Start PSTN Connection
 
 The code snippet below allows you to start a Telephony connection with Symbl via PSTN protocol: 
 
 ```py
 import symbl
 
-phoneNumber = "" #Zoom phone number. Can be found in invitation 
-meetingId = "" # Your zoom meetingId
-password = "" # Your zoom meeting passcode
+phoneNumber = "" #A PSTN number or a Zoom number 
```

Comment:
Zoom is a good example for telephony API, however it's not limited to Zoom only. We can write "A valid US phone number" here. WDYT?

Reply:
👍🏽